root/usr/bin/discovery-menu

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/ruby
+
+require 'discovery/menu'
+require 'discovery/screen/countdown'
+require 'discovery/screen/facts'
+require 'discovery/screen/foreman'
+require 'discovery/screen/network'
+require 'discovery/screen/primary_iface'
+require 'discovery/screen/ssh'
+require 'discovery/screen/status'
+require 'discovery/screen/welcome'
+
+# Disable GC until https://github.com/theforeman/ruby-newt/issues/2 is fixed,
+# see also: http://projects.theforeman.org/issues/11623
+GC.disable
```

Comment:
For the record, there was a problem in the newt cleanup code which I have resolved in https://github.com/theforeman/ruby-newt/commit/5e0144a5dceba285a8d4a516086e0045087294dd but until I will have a chance to do full code review in regard to memory leaks and GC problems, I'd rather keep this disabled for 3.0.0 release. Afterwards let's enable it for nightly builds.

I am tracking this in http://projects.theforeman.org/issues/11623